doc/examples/plot_multiblock_local_binary_pattern.py

```diff
@@ -0,0 +1,82 @@
+"""
+===========================================================
+Multi-Block Local Binary Pattern for texture classification
+===========================================================
+
+In this example, we will see how to compute the multi-block
+local binary pattern at a specified image and how to visualize it.
+
+The features are calculated in a way similar to local binary
+patterns, except that summed up pixel values
+rather than pixel values are used.
+
+`MB-LBP` is an extension of LBP that can be computed on any
+scale in a constant time using integral image. It consists of
+`9` equal-sized rectangles. They are used to compute a feature.
+Sum of pixels' intensity values in each of them are compared
+to the central rectangle and depending on comparison result,
+the feature descriptor is computed.
+
+We will start with a simple image that we will generate
```

Comment:
First, we generate an image to illustrate the functioning of MB-LBP (no backticks required): we take a `(9, 9)` rectangle and divide it into `(3, 3)` block, upon which we then apply MB-LBP.